### PR TITLE
better error message if config file does not exist

### PR DESCRIPTION
**Pull request by LightningHelper**
I assume if one will want to run lnd without a config (which is strange) this push should not be merged

### CODE REVIEW

File: config.go [hunk 1]
```diff
@@ -359,6 +359,11 @@ func loadConfig() (*config, error) {
 			)
 		}
 	}
+	
+	// Check if config file exist
+        if _, err := os.Stat(configFilePath); os.IsNotExist(err) {
```
Comment (Roasbeef) [inline config.go]: A config file isn't required.

Comment (MrManPew) [reply config.go]: Ok, what about now? If a user  supplied a config file from the command line and it does not exist than we should raise an exception

File: config.go [hunk 2]
```diff
@@ -359,6 +359,13 @@ func loadConfig() (*config, error) {
 			)
 		}
 	}
+	
+        // If user supplied alternative config, check if it exist
```
Comment (cfromknecht) [inline config.go:363]:
```suggestion
      // If user supplied alternative config, check if it exists.
```

File: config.go [hunk 3]
```diff
@@ -359,6 +359,13 @@ func loadConfig() (*config, error) {
...
+	if configFilePath != defaultConfigFile {
+		if _, err := os.Stat(configFilePath); os.IsNotExist(err) {
+			return nil, fmt.Errorf("command line config file does not exist in %s", configFilePath)
```
Comment (cfromknecht) [inline config.go:366]: nit: line should be wrapped at 80 chars

Comment (MrManPew) [reply config.go]: fixed

**Comment by cfromknecht** [comment]
@MrManPew the current version fails to compile with this error:
```
Error: ./config.go:500:23: undefined: defaultConfigFile
```

**Comment by MrManPew** [comment]
Maybe now?

**Comment by MrManPew** [comment]
@cfromknecht is that ok? I don't understand this error message:
```
config.go:506: File is not `goimports`-ed (goimports)
```

**Comment by guggero** [comment]
The comment needs to be indented with a tab character, not 4 spaces. You can also locally run `make fmt` (to format) and then `make lint` to check if the linter is happy.

**Comment by MrManPew** [comment]
What now?? I've fixed the lint..

**Comment by guggero** [comment]
Replaced by https://github.com/lightningnetwork/lnd/pull/6080 to speed up review process. Original commit author was preserved, thank you for your contribution.

**Pull request closed without merging**